internal-registry/eclipse/cloud-shell/nightly/meta.yaml

```diff
@@ -24,7 +24,7 @@ spec:
       cookiesAuthEnabled: true
   containers:
   - name: che-machine-exec
-    image: "quay.io/eclipse/che-machine-exec:nightly"
+    image: "${IMAGE_plugin_eclipse_cloud_shell_nightly}"
```

Comment:
It makes sense to extract cloud-shell to env vars as well but that it's mainly useful for local testing.
I suppose for WebTerminal operator we should not deliver it as well as web-terminal-dev. WDYT?

Reply:
We can strip it out but I don't know what we gain except complexity -- we'll need this *eventually* anyways. If we intend this operator to ultimately be used for full workspaces as well as web terminals, we'll end up implementing this within a few months anyways.